src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2294,8 +2436,12 @@ class CgroupUtils(object):
                 if 'systemd' in flags or 'name=systemd' in flags:
                     paths['systemd'] = \
                         self._assemble_path('systemd', entries[1], flags)
-        if not paths:
-            raise CgroupConfigError('Cgroup paths not detected')
+
+#        if some hosts do not have cgroup controllers mounted
+#        don't panic, let main code handle it by doing nothing
+#        if not paths:
+#            raise CgroupConfigError('Cgroup paths not detected')
```

Comment:
Same here I think code that is not needed should be deleted.

Reply:
Changing...